### DISCUSSION TITLE
[Bug]: Problem with path argument for retrieval assistants

### DISCUSSION THREAD

**Discussion by jimgoo** [danny-avila/LibreChat#2247] [Troubleshooting]
### What happened?

I noticed this happens for assistants with retrieval enabled. The message at the end is `Something went wrong. Here's the specific error message we encountered: The "path" argument must be of type string. Received undefined`. 

https://github.com/danny-avila/LibreChat/assets/2735842/f494f7e3-8efe-4833-b561-09ab6f5b5c20

I'm on this commit:

```
commit a00756c469f5c30080aa28605eafcc8bcdb735ef (HEAD -> main, origin/main, origin/HEAD)
Author: Danny Avila <messagedaniel@protonmail.com>
Date:   Fri Mar 29 08:23:38 2024 -0400
```

### Steps to Reproduce

1. Set up an assistant with retrieval enabled. I did this through the OpenAI web interface since I was having trouble doing it through LibreChat. 
2. Attach this file to the assistant: https://raw.githubusercontent.com/danny-avila/LibreChat/main/docs/features/token_usage.md. 
3. Use the system prompt "Answer questions using only the information in the document files available to you. With every response, include the source filename and relevant text as a citation."
4. Now type in LibreChat "How do I manually add balances?"
5. You should get the error `Something went wrong. Here's the specific error message we encountered: The "path" argument must be of type string. Received undefined`.

### What browsers are you seeing the problem on?

Firefox, Chrome

### Relevant log output

```shell
2024-03-29 14:07:14 error: [/assistants/chat/] The "path" argument must be of type string. Received undefined
```

### Screenshots

See video

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Thanks for the report, I can reproduce it with your exact instructions. I have a hunch where the exact issue is.

**Comment by danny-avila** [comment]
Merged a fix here: https://github.com/danny-avila/LibreChat/pull/2248
